[5.1] Add icon for Windows version FFmpeg

nyanmisaka opened the pull request:
**Changes**
- Add icon for Windows version FFmpeg

![ff](https://github.com/jellyfin/jellyfin-ffmpeg/assets/14953024/80a9d173-c8d3-4b9b-a92b-a740b71706de)

Cosmetic changes to make it easier for users to distinguish from vanilla FFmpeg builds.
The icon color comes from https://jellyfin.org/docs/general/contributing/branding

Shadowghost reviewed (approved):
We should add it to the UX repo too..

Might also inlcude it into the readme (we still need to overault it anyway)

nyanmisaka committed:
Fix libxml2 build

Signed-off-by: nyanmisaka <nst799610810@gmail.com>

nyanmisaka committed:
Add icon for Windows version FFmpeg

Cosmetic changes to make it easier for users to distinguish from vanilla FFmpeg builds.

Debian `quilt` uses GNU style patch and it cannot handle patches that contain
binary diff. See https://savannah.nongnu.org/bugs/?55815

Signed-off-by: nyanmisaka <nst799610810@gmail.com>

nyanmisaka commented:
> We should add it to the UX repo too..

Done in https://github.com/jellyfin/jellyfin-ux/pull/56
